src/components/shared/buttons/SubmitButton/SubmitButton.styles.ts

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const Container = styled.div<{ styles: any }>`
+  button {
```

Comment:
Instead of overriding properties with `!important` - could You please a button component from scratch with all styles.